insights/combiners/rhel_for_edge.py

```diff
@@ -0,0 +1,64 @@
+"""
+Combiner for edge computing systems
+===================================
+This combiner uses the following parsers to determine if the system is an edge computing systems.
+
+* :py:class:`insights.parsers.installed_rpms.InstalledRpms`
+* :py:class:`insights.parsers.cmdline.CmdLine`
+* :py:class:`insights.parsers.systemd.unitfiles.ListUnits`
+* :py:class:`insights.parsers.redhat_release.RedhatRelease`
+"""
+from insights.core.plugins import combiner
+from insights.parsers.cmdline import CmdLine
+from insights.parsers.installed_rpms import InstalledRpms
+from insights.parsers.systemd.unitfiles import ListUnits
+from insights.parsers.redhat_release import RedhatRelease
+
+
+@combiner(InstalledRpms, CmdLine, ListUnits, RedhatRelease)
+class RhelForEdge(object):
+    """
+    Combiner for checking if the system is an edge computing systems.
```

Comment:
I think that `Sphinx` will have a problem parsing the comments. Have you tried to generate the docs via `make html_debug`?
Also, I have a problem understanding the docstring; maybe it would be worth reworking it a bit.

Reply:
@jobselko - We checked the docs via `html_debug`. Could you please point out what kind of problem in detail?

Reply:
I am sorry, you are right. For some reason, I had a missing empty line behind `::`, but in the current version, the formatting is ok.